packages/fyndiq-component-alert/src/index.js

```diff
@@ -13,13 +13,17 @@ export default class Alert extends React.Component {
       'bad',
     ]),
     unclosable: PropTypes.bool,
+    componentKey: PropTypes.string,
+    stopShowingWhenClosed: PropTypes.number,
     onClose: PropTypes.func,
   }
 
   static defaultProps = {
     children: '',
     type: 'info',
     unclosable: false,
+    stopShowingWhenClosed: undefined,
+    componentKey: '',
```

Comment:
Since this prop is really tied with `stopShowingWhenClosed`, maybe you should name it accordingly (for instance `stopShowingWhenClosedKey` (which is super long, I agree, but maybe it's better to understand what it really means)